### ISSUE TITLE
web-launcher opens chromium when my default is firefox

### ISSUE THREAD

**Issue by TheTechRobo** [Elive/elive-tools#6]
i've set multiple times my default to firefox, yet eltrans which uses web-launcher ALWAYS opens chromium.

**Comment by Thanatermesis** [comment]
The selection is based on requirement, for example if is a "web app"  (like the Asana one found in your applications menus), is needed / preferred chrome* ones

So the question is more like how much you need it to be firefox instead, and also **how did you configured** firefox to be the default ?

which result is this for you?:

~ ❯❯❯ file /etc/alternatives/x-www-browser
/etc/alternatives/x-www-browser: symbolic link to /usr/bin/google-chrome-stable

**Comment by TheTechRobo** [comment]
`/etc/alternatives/x-www-browser: symbolic link to /usr/bin/opera`, yet it still loads chromium. :baffled:

**Comment by Thanatermesis** [comment]
Yeah so then the reason is that chromium is selected as preference because it includes some features that are not available on firefox, like "app" mode or similar, what are you trying to run exactly ?

**Comment by TheTechRobo** [comment]
The mind maps program opens chromium

eltrans opens chromium (google translate)
